PopPUNK/__main__.py

```diff
@@ -872,9 +863,34 @@ def assign_query(dbFuncs, ref_db, q_files, output, update_db, full_db, distances
         # QC distance matrix
         qcPass = qcDistMat(distMat, refList, queryList, max_a_dist)
 
-        # Assign to strains or lineages, as requested
+        # Assign to strains or lineages, as requested.
+        # Both need the previous model loaded
+        model_prefix = ref_db
+        if model_dir is not None:
+            model_prefix = model_dir
+        model_file = model_prefix + "/" + os.path.basename(model_prefix)
+
```

Comment:
Hit an error when querying - first problem looks to be that each model is saved with the rank number at the end, but model file loading does not append the rank number.